plejd/Dockerfile

```diff
@@ -1,7 +1,11 @@
-ARG BUILD_FROM
+ARG BUILD_FROM=ghcr.io/hassio-addons/base/amd64:18.1.4
```

Comment:
This refers to a specific architecture (amd64), which I don't think is desirable. Perhaps need to use BUILD_ARCH? (Although from the docs, if building from a fixed image, it looks like it might be preferred to put the specific image in build.yaml instead - there was previously a build.json which I removed in my initial commit. It's unclear to me in which circumstances it makes a difference putting the image name/version here vs in build.yaml)